Galaxus.Functional.NUnit/Galaxus.Functional.NUnit.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <LangVersion>latest</LangVersion>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>true</IsPackable>
+
+        <RootNamespace>Galaxus.Functional.NUnit</RootNamespace>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
```

Comment:
Didn't notice it at first, but since it's net2.0 now it gave me a warning on restore - why include all these packages? The only one we truly need is NUnit, the rest is for test projects.